src/MIDebugEngine/Engine.Impl/DebuggedProcess.cs

```diff
@@ -384,9 +384,15 @@ public DebuggedProcess(bool bLaunched, LaunchOptions launchOptions, ISampleEngin
             ErrorEvent += delegate (object o, EventArgs args)
             {
                 // NOTE: Exceptions leaked from this method may cause VS to crash, be careful
-
                 ResultEventArgs result = (ResultEventArgs)args;
-                _callback.OnError(result.Results.FindString("msg"));
+                string msgString = "msg";
```

Comment:
I suggest TryFind "msg" first and then for "message" (if LLDB) so that this doesn't fail in case they ever "fix" the issue on the LLDB side.